ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -656,14 +656,30 @@ static KsqlRestApplication buildApplication(
 
     final String commandTopicName = ReservedInternalTopics.commandTopic(ksqlConfig);
 
+    CommandTopicBackup commandTopicBackup = new CommandTopicBackupNoOp();
+    if (ksqlConfig.getBoolean(KsqlConfig.KSQL_ENABLE_METASTORE_BACKUP)) {
+      if (ksqlConfig.getString(KsqlConfig.KSQL_METASTORE_BACKUP_LOCATION).isEmpty()) {
```

Comment:
can we condense these 2 into 1 config? if the backup location is set then just enable backups. Also why do we ever want to disable backups?

Reply:
I've combined the configs now. @spena do you know the scenario for turning off backups?